docs/yourbase-ruby.md

```diff
@@ -0,0 +1,82 @@
+# YourBase Ruby Acceleration
+
+Tests are important. For large monoliths, they're also a major source of drag on velocity.
+
+YourBase is a tool that traces your tests to determine which files they depend on. It uses this information to determine which tests must run, because they are new or have changed dependencies. Tests without changed dependencies are skipped.
+
+YourBase works with Ruby versions >= 2.3 and RSpec 3+.
+
+## Getting Started
+1. Add `gem 'yourbase-rspec', '~> 0.5.6',` to your Gemfile. If you have a `:test` group, add it there.
+2. `bundle install` from the command line.
+3. If you are not in a Rails project, you will also need to `require 'yourbase-rspec` your spec_helper (or at the top of the spec file you want to run).
+
+Once you have a token, simply follow the steps below:
+```sh
+bundle add yourbase-rspec --git "https://${YOURBASE_DOWNLOAD_TOKEN?}:x-oauth-basic@github.com/yourbase/yourbase-rspec-skipper-engine.git" && bundle install
+```
```

Comment:
sorry if I missed this in the first pass, but I think the 1,2,3 and the ```sh``` section below it assume different things about the the public/privateness of the repo.

If we're going to publish this today-ish for Gusto purposes, we should make step `1.` include the token syntax,
`gem 'yourbase-rspec', '~> 0.6.0, git: "https://yourbase-tokenbot:#{ENV.fetch(YOURBASE_DOWNLOAD_TOKEN){nil}}@github.com/yourbase/yourbase-rspec-skipper-engine", branch: 'main'`, and then lines 14-16 are redundant and can be removed.